File: 프로그래머스/2/42586. 기능개발/기능개발.js
function solution(progresses, speeds) {
    const days = [];
    progresses.map((v,i) =>
        days.push(Math.ceil((100-v)/speeds[i]))
    );
    console.log(days);
    
    const ans = [0];
    let count = 0;
    let stdday = days[0];
    const pl = progresses.length;
    for (let i=0, j=0; i<pl; i++) {
        if (days[i]<=stdday) {
            ans[j]+=1;
        }
        else {
            ans[++j]=1;
            stdday = days[i];
        }
    }
    return ans;
}

File: 프로그래머스/2/42586. 기능개발/기능개발.py
from collections import deque
import math
def solution(progresses, speeds):
    q = deque()
    for i in range(len(progresses)):
        q.append(math.ceil((100-progresses[i])/speeds[i]))
    
    cnt = 0
    standard = q[0]
    answer = []
    
    while q:
        num = q.popleft()
        if num>standard:
            answer.append(cnt)
            standard = num
            cnt = 1
        else:
            cnt+=1
    answer.append(cnt)
    
    return answer